Refresh.GameServer/Endpoints/ApiV3/AuthenticationApiEndpoints.cs

```diff
@@ -262,8 +262,9 @@ public ApiOkResponse DenyVerificationRequest(RequestContext context, GameDatabas
         {
             database.AddRegistrationToQueue(body.Username, body.EmailAddress, passwordBcrypt);
             return new ApiAuthenticationError(
-                "Your account has been created, but it is not yet activated. " +
-                "To complete registration, simply log in from LBP and your new account will be activated.", true);
+                "Your account has been put into the registration queue, but it is not yet activated. " +
+                "To complete registration, simply log in from LBP within the next hour and your new account will be activated. " +
```

Comment:
Something ive learned helping people connect, they dont understand the phrase "log in", perhaps it should be _something_ along the lines of "To complete registration, patch LBP, and then start browsing community levels. or your registration will be deleted in 1 hour", where it explains that "log in" means "start browsing community levels/access ingame online features". the words "login" isnt really used by the game to explain the connection to end users

Reply:
![image](https://github.com/LittleBigRefresh/Refresh/assets/40577357/bb0d3807-165d-4b67-8e22-8bda1e79a2fd)
How's this?

Reply:
should be much better